PontoCom/View/LoginView.swift

```diff
@@ -9,7 +9,8 @@ import SwiftUI
 import FirebaseAuth
 
 struct LoginView: View {
-    @StateObject private var viewModel = LoginViewModel()
+        
+    @ObservedObject var viewModel = LoginViewModel()
```

Comment:
Melhor deixar assim, não dá erro

```suggestion
    @StateObject private var viewModel = LoginViewModel()
```